pages/index.tsx

```diff
@@ -1,71 +1,18 @@
-import type { NextPage } from 'next'
-import Head from 'next/head'
-import styles from '../styles/Home.module.css'
+import type { NextPage } from "next";
+import Head from "next/head";
+import { Footer, Header } from "@components";
 
 const Home: NextPage = () => {
   return (
-    <div className={styles.container}>
+    <div style={{ height: "800px" }}>
```

Comment:
is this necessary ?

Reply:
nop, it was juste for testing the layout ahah